packages/schemas/src/types/index.ts

```diff
@@ -11,7 +11,6 @@ export * from './role.js';
 export * from './verification-code.js';
 export * from './application.js';
 export * from './system.js';
-export * from './tenant.js';
```

Comment:
I don't quite get the changes made in `seeds/tenant.ts` and `types/index.ts`

Reply:
`types/index.ts` is like API types, which will be used by logto/core and logto/console. We migrate tenant-related API types (in our case, `TenantInfo`) and put them along with the initialization of the tenant model to avoid them being imported by logto/console (this could lead to some parcel errors).